packages/api/public/spec/schemas/language.yml

```diff
@@ -2,15 +2,15 @@ allOf:
   - $ref: './create-language.yml'
   - type: object
     properties:
-      code:
```

Comment:
Why did these get removed?

Reply:
Because I moved them to `create-language.yml`. Now `language.yml` extends `create-language.yml`